### ISSUE TITLE
Improve navigation

### ISSUE THREAD

**Issue by sebtesobe** [OpenBankProject/API-Explorer#23]
- Make the logo, login and dropdown navigation fixed. 
- When scrolling down, make them smaller and hide catalog description

**Comment by sebtesobe** [comment]
Please have a look locally or on https://apiexplorersandbox.openbankproject.com/

**Comment by simonredfern** [comment]
Very nice. I'm even wondering if we need the initial state or could just use the scroll state and show "popup" messages (via Liftweb) when we select another catalog or version etc. (Or otherwise highlight the version we are on etc.) Closing for now so we can use it a bit.

**Comment by simonredfern** [comment]
One thing - the anchor navigation from left panel sometimes misses its target. 

to reproduce click the endpoints in the left bar and see if you are taken to the endpoint correctly.

**Comment by sebtesobe** [comment]
I changed the implementation a bit, but it is still buggy. Previously, the first click in the left panel hit the target, but all subsequent clicks missed. Now the first click in the left panel will miss the target, but all subsequent clicks will hit. (A side effect of the current implementation is the blue description box, which is hidden on scroll, has now a fixed height) It seems that the browser calculates the position where to jump to before the javascript which is executed on scroll and which will redefine paddings.

I believe it would be easier if the height of the navigation was not changing on scroll. Can we have only one version of the top navigation which is maybe smaller than the default / non-scroll version, but larger than the scroll version?

https://apiexplorersandbox.openbankproject.com has been updated.
